source/extensions/Sekai.Interop.Common/FuncLoader.Desktop.cs

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) The Vignette Authors
+// Licensed under MIT. See LICENSE for details.
+
+// Contains code from MonoGame.
+// Copyright 2021 the MonoGame Team.
+// Licensed under the Microsoft Public License and MIT.
+// See https://github.com/MonoGame/MonoGame/blob/develop/LICENSE.txt.
+using System.Runtime.InteropServices;
+
+namespace Sekai.Interop.Common;
+public class FuncLoader
```

Comment:
[System.Runtime.InteropServices.NativeLibrary](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.nativelibrary?view=net-6.0) seems to do what is written here

Reply:
Lemme see if I can reimplement some of the methods using NativeLibrary. At best when I checked this, this seems to do some stuff better than `NativeLibrary` implementation wise.